index.js

```diff
@@ -46,6 +48,29 @@ export const provideLinter = () => ({
 	lint
 });
 
-export const activate = () => {
+export function activate() {
 	require('atom-package-deps').install();
-};
+	this.subscriptions = new CompositeDisposable();
+	this.subscriptions.add(atom.commands.add('atom-text-editor', {
+		'linter-xo:fix-file': function fixFile() {
+			const textEditor = atom.workspace.getActiveTextEditor();
+			if (!textEditor || textEditor.isModified()) {
+				// Ignore invalid or unsaved text editors
+				return;
+			}
+			const filePath = textEditor.getPath();
+			const fileDir = path.dirname(filePath);
+			new BufferedNodeProcess({
```

Comment:
Any reason you're spawning the CLI instead of using the API? I know the API is not documented and I'm happy to do the changes. Just wanted to ask if there's a particular reason it's done this way.